main.c

```diff
@@ -238,22 +387,38 @@ static void can_msg_handler(const can_msg_t *msg) {
         return;
     }
 
+    // make able to handle multiple actuator
     switch (msg_type) {
         case MSG_GENERAL_CMD:
             cmd_type = get_general_cmd_type(msg);
             if (cmd_type == BUS_DOWN_WARNING) {
```

Comment:
Not sure why this code ever existed but we should only ever be changing actuators in response to error conditions or actuator commands, not a bus down warning. Just ignore this message.